lib/repl.js

```diff
@@ -698,6 +699,10 @@ function filteredOwnPropertyNames(obj) {
   return Object.getOwnPropertyNames(obj).filter(intFilter);
 }
 
+REPLServer.prototype.complete = function() {
```

Comment:
Not really related to your PR, @diosney, but just to follow up on my comment. I've noticed this in a number of places - methods attached to an object's prototype that are ostensibly public, but undocumented. In my view, this is not ideal and should be rectified by deciding what is intended to be public and what is not, and structuring the code and modifying the docs so they reflect the true intent.

@Trott we spoke briefly about issues similar to this (e.g. `someObj._someSemiPrivateProperty`) during onboarding. I briefly looked through the issues but could not find anything that addressed this as an overarching concern. If there is some place where this is being discussed, can you point me in the right direction? If not, would it be reasonable to start this discussion with an issue on nodejs/node?